core/capabilities/ccip/configs/solana/chain_writer.go

```diff
@@ -323,6 +341,92 @@ func getRouterAccountConfig(routerProgramAddress string) chainwriter.PDALookups
 	}
 }
 
+func getAddressConstant(address string) chainwriter.AccountConstant {
+	return chainwriter.AccountConstant{
+		Address:    address,
+		IsSigner:   false,
+		IsWritable: false,
+	}
+}
+
+func getFeeQuoterConfig(offrampProgramAddress string) chainwriter.PDALookups {
+	return chainwriter.PDALookups{
+		Name:      "FeeQuoter",
```

Comment:
nit: there are consts for many of these contracts: https://github.com/smartcontractkit/chainlink-ccip/blob/d25c9c136c0d5ecac78872d5c6db4223be79f905/pkg/consts/consts.go#L5